### DISCUSSION TITLE
Who is Sergey Popov?

### DISCUSSION THREAD

**Discussion by mihalyr** [community/community#59432] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Dear GitHub,

Why does Copilot suggest me a comment that my new class was created by someone called Sergey Popov in 2016? 

![image](https://github.com/community/community/assets/26523902/469ca349-aa26-43d7-a26b-9c08fc25a102)

I have Copilot for Business and I do have public code blocked explicitly. Does that "Blocked" option work properly?

![image](https://github.com/community/community/assets/26523902/49a16b71-cb11-472d-b381-6d3364d4e672)

Regards,
Robert

**Comment by Akash1134** [comment, marked as answer]
Hi @mihalyr ,

I have shared this feedback with my team internally. Thanks for reporting and sharing.
